src/payment/strategies/stripev3/stripev3-payment-strategy.ts

```diff
@@ -0,0 +1,129 @@
+import { CheckoutStore, InternalCheckoutSelectors } from '../../../checkout';
+import {
+    InvalidArgumentError,
+    MissingDataError,
+    MissingDataErrorType,
+    StandardError
+} from '../../../common/error/errors';
+import { OrderActionCreator, OrderRequestBody } from '../../../order';
+import { OrderFinalizationNotRequiredError } from '../../../order/errors';
+import { PaymentArgumentInvalidError } from '../../errors';
+import PaymentActionCreator from '../../payment-action-creator';
+import PaymentMethodActionCreator from '../../payment-method-action-creator';
+import { PaymentInitializeOptions, PaymentRequestOptions } from '../../payment-request-options';
+import PaymentStrategy from '../payment-strategy';
+
+import {
+    StripeCardElement,
+    StripeResponse,
+    StripeScriptLoader,
+    StripeV3Js
+} from './index';
+
+export default class StripeV3PaymentStrategy implements PaymentStrategy {
+    private stripeJs?: StripeV3Js;
+    private cardElement?: StripeCardElement;
+
+    constructor(
+        private _store: CheckoutStore,
+        private _paymentMethodActionCreator: PaymentMethodActionCreator,
+        private _paymentActionCreator: PaymentActionCreator,
+        private _orderActionCreator: OrderActionCreator,
+        private _stripeScriptLoader: StripeScriptLoader
+    ) {}
+
+    initialize(options: PaymentInitializeOptions): Promise<InternalCheckoutSelectors> {
+        const DOMElement = options.stripev3;
+
+        if (!DOMElement) {
+            throw new InvalidArgumentError('Unable to initialize payment because "options.stripe" argument is not provided.');
+        }
+
+        const paymentMethod = this._store.getState().paymentMethods.getPaymentMethod(options.methodId);
+
+        if (!paymentMethod) {
+            throw new MissingDataError(MissingDataErrorType.MissingPaymentMethod);
+        }
+
+        return this._stripeScriptLoader.load(paymentMethod.initializationData.stripePublishableKey)
+            .then((stripeJs: StripeV3Js) => {
+                this.stripeJs = stripeJs;
+                const elements = this.stripeJs.elements();
+                this.cardElement = elements.create('card', {
+                    style: DOMElement.elementProps,
+                });
+
+                if (!this.cardElement) {
```

Comment:
`create` method actually always returns `StripeCardElement` so you don't actually need to do a null check here. If you just define a variable like this, you should be able to use it directly.

```ts
const cardElement = elements.create('card', {
    style: stripeOptions.style,
});

cardElement.mount(`#${stripeOptions.containerId}`);
```

If you are planning to use `cardElement` in another method, you can store it as an instance member afterwards. i.e.: 

```ts
this._cardElement = cardElement;
```